src/client/components/Button/ButtonComponent.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ButtonComponent.styles.css';
+
+export default function ButtonComponent({ title, backgroundColor, onClick }) {
+  return (
+    <div className="button-component">
+      <button
+        type="button"
+        onClick={onClick}
+        className="button button--primary button--medium"
```

Comment:
As you have it now, you are using 3 different classes for this button, and therefore overwritting some properties. Try to revisit which classes and properties you actually need

Reply:
okay.. thanks .. i will remove extra classes and keep one